src/games/strategy/triplea/delegate/BattleTracker.java

```diff
@@ -566,7 +575,9 @@ public void takeOver(final Territory territory, final PlayerID id, final IDelega
       final PlayerAttachment paWhoseCapital = PlayerAttachment.get(whoseCapital);
       final List<Territory> capitalsList =
           new ArrayList<Territory>(TerritoryAttachment.getAllCurrentlyOwnedCapitals(whoseCapital, data));
-      if (paWhoseCapital != null && paWhoseCapital.getRetainCapitalNumber() < capitalsList.size()) // we are losing one right now, so it is
+      if (paWhoseCapital != null && paWhoseCapital.getRetainCapitalNumber() < capitalsList.size()) // we are losing one
```

Comment:
bad formatting, you're responsible for good formatting even if the auto-formatter does not do the right thing. IE: I made the change because that is what my formatter did is both missing the point of formatting and also not valid. When mass formatting all files, I think we can let things slide, but for a single file in a controlled change, either the formatting should be good or not changed IMO.